app/models/srpm.rb

```diff
@@ -39,8 +39,13 @@ class Srpm < ApplicationRecord
            primary_key: 'name',
            foreign_key: 'repo' # dependent: :destroy
 
-  validates :groupname, presence: true
+  multisearchable against: [:name, :summary, :description, :filename, :url]
+
+  pg_search_scope :query,
+                  against: %i(name summary description filename url),
+                  using: { tsearch: { prefix: true}}
```

Comment:
Move this after `after_destroy` callback.

Use `[]` for arrays.

Reply:
should `pg_scopes` be with the scope group, or be in another place apart it?